docs/base/soa.md

```diff
@@ -0,0 +1,87 @@
+# Configuracions sincronització SOA
+
+Documentació general del mòdul de sincronització mitjançant fitxers SOA.
+
+## Menús
+
+El mòdul de sincronització proveeix dos llistats ubicats a "**_OpenERP Menu >
+Sync_**":
+
+- [Fitxers Importats](#fitxers-importats)
+- [Configuració](#configuracio)
+
+![](_static/soa/oerp_menu_soa.png)
+
+### Fitxers Importats
+
+En aquest menú es mostra un llistat dels fitxers de sincronització (SOA) importats.
+Des del llistat podem veure:
+
+- Data de la importació
+- Nom del fitxer importat
+- Tipus de sincrontizació:
+  - Sincronització massiva: Importació de moltes actualitzacions
+  - Sincronització immediata: Importació d'una sola actualització
+- Total de línies importades
+- Total de línies correctes
+- Total de línies amb error
+- Estat de la importació
+- Missatge descriptiu amb l'estat de la importació
+
+!!! Note "Nota"
+    Si accedim a un d'aquests registres, podrem trobar com a adjunt el fitxer
+    de sincronització importat.
+
+![](_static/soa/soa_llistat.png)
+
+### Configuració
+
+En aquest menú es mostren les configuracions per la importació dels fitxers de
+sincrontizació (SOA) i de la notificació dels resultats d'aquesta importació.
+
+Podem trobar les següents columnes:
+
+- Ordre en que es mostren al llistat
+- Mòdul al que pertany la configuració,
+  únicament hauria d'haver "giscesoa_sync" en aquest llistat.
+- Clau utilitzada per l'ERP
+- Valor utilitzar per l'ERP per una clau concreta
+- Descripció de com ha de ser el valor per una clau concreta
+
+![](_static/soa/soa_configuracio.png)
+
+#### Configuracions SOA
+
+En el fitxer de configuracions hi ha les següents configuracions:
+
+| Clau | Descripció |
+|-|-|
+| `configurat` | "1" vol dir que el mòdul ja s'ha configurat; "0" és el valor per defecte per assegurar que el mòdul no nomes s'ha instal.lat sino que a mes a mes s'ha configurat. |
+| `sufix_massiva` | Extensió que s'espera pels fitxers d'intercanvi per sincronització massiva. |
+| `sufix_immediata` | Extensió que s'espera pels fitxers d'intercanvi per sincronització immediata. |
+| `sufix_fitxer_duplicat` | Sufix que s'afegira quan es trobi al directori d'entrada un fitxer amb un nom que ja s'havia processat. |
+| `sufix_correctes` | Extensió dels fitxers que han de contenir les linies que s'han processat correctament. |
+| `sufix_errors` | Extensió dels fitxers que han de contenir les línies que s'han intentat processat sense èxit. |
```

Comment:
Write the same in both languages:

(ca) `Extensió dels fitxers que han de contenir les línies que s'han intentat processat sense èxit.`
(es) `Extensión de los ficheros que deben contener las líneas que se han procesado sin éxito.`

`processat` → `processar`
(es) `Extensión de los ficheros que deben contener las líneas que se han intentado procesar sin éxito.`